clients/client-python/gravitino/filesystem/gvfs_base_operations.py

```diff
@@ -121,6 +123,14 @@ def __init__(
         self._catalog_cache = LRUCache(maxsize=100)
         self._catalog_cache_lock = rwlock.RWLockFair()
 
+        self._enable_credential_vending = (
+            False
+            if options is None
+            else options.get(
+                GVFSConfig.GVFS_FILESYSTEM_ENABLE_CREDENTIAL_VENDING,
+                self.ENABLE_CREDENTIAL_VENDING_DEFAULT,
+            )
+        )
```

Comment:
This code is difficult to parse ... why don't we use just `if ... else`?

Reply:
The current writing style is the commonly used concise style in Python.